tools/python_api/py_query_result.cpp

```diff
@@ -35,8 +38,15 @@ py::list PyQueryResult::getNext() {
     return move(result);
 }
 
-void PyQueryResult::writeToCSV(py::str filename) {
-    queryResult->writeToCSV(filename);
+void PyQueryResult::writeToCSV(
```

Comment:
Can you update our python API doc as well? Here is the link: https://github.com/kuzudb/kuzudb.github.io/tree/main